src/js/pages/jobs/JobDetailPage.js

```diff
@@ -97,7 +103,7 @@ class JobDetailPage extends mixin(StoreMixin, TabsMixin) {
       <button
         className="button button-inverse button-stroke"
         key="edit"
-        onClick={this.handleEditButtonClick.bind(this, jobID)}>
+        onClick={this.handleEditButtonClick.bind(this)}>
```

Comment:
As this is a actually small change - isn't it? - I would prefer to have this in this PR.
But decide on your own. You could also create overhead and create a new PR.